POPS & Lipid II updates

pstansfeld opened the pull request.

pstansfeld committed:
Update gro.py

pstansfeld committed:
Update gro.py

pstansfeld committed:
Update gro.py

pstansfeld committed:
Update gro.py

pstansfeld committed:
Update gro.py

pstansfeld committed:
Update gro.py

pstansfeld committed:
Update CARD.itp

pstansfeld committed:
Merge branch 'owenvickery:master' into master

chelsea-brown committed:
Add files via upload

chelsea-brown committed:
Create lip2.top

chelsea-brown committed:
Rename database/fragments/martini_3-0_charmm36/non_protein/LIP2.itp to database/fragments/martini_3-0_charmm36/non_protein/LIP2/LIP2.itp

chelsea-brown committed:
Rename database/fragments/martini_3-0_charmm36/non_protein/LIP2.pdb to database/fragments/martini_3-0_charmm36/non_protein/LIP2/LIP2.pdb

chelsea-brown committed:
Rename database/fragments/martini_3-0_charmm36/non_protein/LIP2.top to database/fragments/martini_3-0_charmm36/non_protein/LIP2/LIP2.top

chelsea-brown committed:
Rename database/fragments/martini_3-0_charmm36/non_protein/LIP2_posres.itp to database/fragments/martini_3-0_charmm36/non_protein/LIP2/LIP2_posres.itp

chelsea-brown committed:
Delete lip2.top

pstansfeld committed:
Update LIP2.pdb

robincorey committed:
adding POPS to M3

owenvickery reviewed (changes requested):
I'm not merging these my version as it breaks the absolute path of the top file. Everything is written using absolute so you don't have to copy the forcefield around.

pstansfeld committed:
Create test

pstansfeld committed:
Add files via upload

pstansfeld committed:
Delete test

pstansfeld committed:
Update UDP1.pdb

pstansfeld committed:
Merge branch 'owenvickery:master' into master

pstansfeld committed:
Update README.md

pstansfeld committed:
Delete database/forcefields/amber14_lipid17_will_be_removed.ff directory

pstansfeld committed:
Delete database/forcefields/test_charmm36-mar2019-updated.ff directory

pstansfeld committed:
Delete database/forcefields/charmm36-mar2019-updated.ff directory

pstansfeld committed:
Delete database/fragments/martini_3_beta_charmm36 directory

pstansfeld committed:
Delete database/fragments/martini_2-2_lipid17_port_use_at_own_risk directory

pstansfeld committed:
Delete database/fragments/martini_2-0_charmm36 directory

pstansfeld committed:
new file:   database/fragments/martini_3-0_charmm36/non_protein/LIP3/LIP3.itp
	new file:   database/fragments/martini_3-0_charmm36/non_protein/LIP3/LIP3.pdb
	new file:   database/fragments/martini_3-0_charmm36/non_protein/LIP3/LIP3.top
	new file:   database/fragments/martini_3-0_charmm36/non_protein/LIP3/LIP3_posres.itp
	modified:   database/fragments/martini_2-2_charmm36/protein_modified/_others/LYS.pdb

pstansfeld committed:
modified:   database/fragments/martini_3-0_charmm36/non_protein/LIP3/LIP3.pdb

Commit:
modified:   database/forcefields/charmm36-jul2020-updated.ff/ffbonded.itp
	modified:   database/fragments/martini_3-0_charmm36/non_protein/LIP3/LIP3.pdb

Commit:
new file:   database/fragments/martini_3-0_charmm36/non_protein/APG/APG.itp

pstansfeld committed:
Create test

pstansfeld committed:
Add files via upload

pstansfeld committed:
Delete database/fragments/martini_2-2_charmm36/non_protein/KLA/test

pstansfeld committed:
Create KLA.itp

pstansfeld committed:
Add files via upload

pstansfeld committed:
Delete database/fragments/martini_2-2_charmm36/non_protein/KLA directory

pstansfeld committed:
Create DPG3.itp

pstansfeld committed:
Add files via upload

pstansfeld committed:
DOPS added

pstansfeld committed:
Merge branch 'master' of https://github.com/pstansfeld/cg2at
merge

pstansfeld committed:
DOPS added

pstansfeld committed:
Delete database/fragments/martini_2-2_charmm36/protein_modified/_others/lys.pdb

Commit:
PI lipids updated with groups

Commit:
PI lipids updated with groups

Commit:
PI lipids updated with groups

pstansfeld committed:
Update README.md

pstansfeld committed:
PI edits
